.gitmodules

```diff
@@ -73,3 +73,7 @@ path = repos/SSSL-DNS
 url = https://github.com/PretendoNetwork/SSSL-DNS.git
 branch = master
 ignore = dirty
+
+[submodule "repos/splatoon"]
+	path = repos/splatoon
```

Comment:
For consistency, please change the formatting so it matches the rest of the file and add the specified branch and ignore options the others have.